apps/web/src/modules/notification/components/NoticardContainer/components/NotiCard/Components/NotiCardHeader/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { NotificationType, UserType } from '@modela/dtos'
+import { Typography } from '@mui/material'
+import { formatDate, formatTime } from 'common/utils/formatter'
+import React from 'react'
+
+import {
+  actorHeaderMap,
+  actorIconMap,
+  castingHeaderMap,
+  castingIconMap,
+} from './constants'
+import { HeaderContainer, IconContainer } from './styled'
+import { HeaderProps } from './types'
+
+const NotiCardHeader = (props: HeaderProps) => {
+  const {
+    type,
+    title,
+    companyName,
+    timestamp,
+    userType,
+    firstName,
+    middleName,
+    lastName,
+  } = props
+  return (
+    <HeaderContainer>
+      <div style={{ display: 'flex', alignItems: 'center' }}>
+        <IconContainer>
+          {userType === UserType.ACTOR
+            ? actorIconMap[type]
+            : castingIconMap[type]}
+        </IconContainer>
+        <div>
+          <Typography variant="h6" sx={{ wordBreak: 'break-word' }}>
+            {userType === UserType.ACTOR
+              ? actorHeaderMap[type]
+              : castingHeaderMap[type]}
+            {userType === UserType.CASTING &&
+            (type === NotificationType.APPROVE_REFUND ||
+              type === NotificationType.REJECT_REFUND)
```

Comment:
Repetitive condition checking to line 46, maybe making it as an another variable ?